File: ProyectoMysql/server/EntityLayer/Mercaderia/MercaderiaMainModel.py
from datetime import datetime
from pydantic import BaseModel
from Utilidades.Enumerado.ProcessActionEnum import ProcessActionEnum

class MercaderiaMainModel:
    MercaderiaId: int 
    Codigo: str 
    NomCategoria: int 
    NomTipoProducto: int 
    NomMarca: int 
    NomModelo: int 
    Descripcion: str 
    UnidadMedidaId: int 
    FechaRegistro: datetime 
    CodUsuario: str 
    EstadoRegistro: bool 
    
    def Cargar(_DB):
        c =  MercaderiaMainModel()
        c.MercaderiaId = _DB["MercaderiaId"] 
        c.Codigo = _DB["Codigo"] 
        c.NomCategoria = _DB["NomCategoria"] 
        c.NomTipoProducto = _DB["NomTipoProducto"] 
        c.NomMarca = _DB["NomMarca"] 
        c.NomModelo = _DB["NomModelo"] 
        c.Descripcion = _DB["Descripcion"] 
        c.UnidadMedidaId = _DB["UnidadMedidaId"] 
        c.FechaRegistro = _DB["FechaRegistro"] 
        c.CodUsuario = _DB["CodUsuario"] 
        c.EstadoRegistro = bool(ord(_DB["EstadoRegistro"])) 
        return c
    

File: ProyectoMysql/Api/LogisticStorage/LogisticStorage.Server/Model/Mercaderia/MercaderiaMainModel.cs
using LogisticStorage.EntityLayer;
using System.Text.Json.Serialization;

namespace LogisticStorage.Server
{
    public class MercaderiaMainModel
    {
        public MercaderiaMainModel()
        {
            this.MercaderiaId = 0;
            this.Codigo = String.Empty;
            this.NomCategoria = String.Empty;
            this.NomTipoProducto = String.Empty;
            this.NomMarca = String.Empty;
            this.NomModelo = String.Empty;
            this.Descripcion = String.Empty;
            this.NomUnidadMedida = String.Empty;
            this.FechaRegistro = DateTime.Now;
            this.CodUsuario = String.Empty;
            this.EstadoRegistro = false;
        }

        public MercaderiaMainModel(MercaderiaEntity Item)
        {
            this.MercaderiaId = Item.MercaderiaId;
            this.Codigo = Item.Codigo;
            this.NomCategoria = Item.NomCategoria;
            this.NomTipoProducto = Item.NomTipoProducto;
            this.NomMarca = Item.NomMarca;
            this.NomModelo = Item.NomModelo;
            this.Descripcion = Item.Descripcion;
            this.NomUnidadMedida = Item.NomUnidadMedida;
            this.FechaRegistro = Item.FechaRegistro;
            this.CodUsuario = Item.CodUsuario;
            this.EstadoRegistro = Item.EstadoRegistro;
        }

        [JsonPropertyName("MercaderiaId")] public int MercaderiaId { get; set; }
        [JsonPropertyName("Codigo")] public String Codigo { get; set; }
        [JsonPropertyName("NomCategoria")] public String NomCategoria { get; set; }
        [JsonPropertyName("NomTipoProducto")] public String NomTipoProducto { get; set; }
        [JsonPropertyName("NomMarca")] public String NomMarca { get; set; }
        [JsonPropertyName("NomModelo")] public String NomModelo { get; set; }
        [JsonPropertyName("Descripcion")] public String Descripcion { get; set; }
        [JsonPropertyName("NomUnidadMedida")] public String NomUnidadMedida { get; set; }
        [JsonPropertyName("FechaRegistro")] public DateTime FechaRegistro { get; set; }
        [JsonPropertyName("CodUsuario")] public String CodUsuario { get; set; }
        [JsonPropertyName("EstadoRegistro")] public Boolean EstadoRegistro { get; set; }
    }
}
